src/client/src/routes/OrderTicketRoute/MediaRecorderComponent.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react'
+
+import MediaRecorder, { BlobEvent, MediaRecorderInterface } from './MediaRecorder'
+import { Timer } from './Timer'
+
+export const mediaRecorderEvents = ['start', 'stop', 'dataavailable', 'pause', 'resume', 'error']
+
+export { BlobEvent }
+
+export interface Props {
+  input: MediaStream
+  mimeType?: string
+  bitsPerSecond?: number
+  rate?: number
+  requestData?: boolean
+  onStart?: (e: Event) => any
+  onStop?: (e: Event) => any
+  onDataAvailable?: (e: BlobEvent) => any
+  onPause?: (e: Event) => any
+  onResume?: (e: Event) => any
+  onError?: (e: Event) => any
+}
+
+interface State {
+  input?: MediaStream
+  recorder?: MediaRecorderInterface
+
+  recording: boolean
+}
+
+class MediaRecorderComponent extends React.PureComponent<Props, any> {
+  static defaultProps = {
+    rate: 256,
+    mimeType: 'audio/webm',
+    bitsPerSecond: 128 * 1000,
+  }
+
+  state: State = {
+    recorder: null,
```

Comment:
If you wanted you could do `recorder: new MediaRecorder(input, { mimeType, bitsPerSecond })` here. (Using this.props.input, etc.)